Game/Source/GDKTestGymsFunctionalTests/GDKTestGymsFunctionalTests.cpp

```diff
@@ -2,8 +2,20 @@
 
 #include "GDKTestGymsFunctionalTests.h"
 #include "LogGymsSpatialFunctionalTest.h"
-#include "Modules/ModuleManager.h"
+#include "TestMapGeneration.h"
 
 DEFINE_LOG_CATEGORY(LogGymsSpatialFunctionalTest);
 
-IMPLEMENT_PRIMARY_GAME_MODULE( FDefaultGameModuleImpl, GDKTestGymsFunctionalTests, "GDKTestGymsFunctionalTests" );
+IMPLEMENT_GAME_MODULE(FGDKTestGymsFunctionalTestsModule, GDKTestGymsFunctionalTests)
+
+#define LOCTEXT_NAMESPACE "FGDKTestGymsFunctionalTestsModule"
+
+void FGDKTestGymsFunctionalTestsModule::StartupModule()
```

Comment:
I'm not sure if logically this is the right module; I think this one is used to store functional tests that depend on GAS which we didn't want to depend on in the GDK plugin. Would it be better to put this into the `FGDKTestGymsModule` instead?

Reply:
Problem there is that the GDKTestGymsModule.build file would need to include the SpatialGDKFunctionalTests module... which maybe is ok but... we have a module inside the TestGyms that already includes SpatialGDKFunctionalTests: DKTestGymsFunctionalTestsModule